src/C4SoundSystem.cpp

```diff
@@ -29,6 +29,8 @@
 #include <utility>
 #include <vector>
 
+const std::string C4SoundSystem::fileExtensions[] = { ".wav", ".ogg" };
```

Comment:
Can be either `constexpr` with const char * or `std::array<std::string, 2 >`. Also, according to style guide, no `=` is needed here.